docs/Automation.md

```diff
@@ -188,78 +201,39 @@ Sample `C#` code—this is interactive, but yours doesn’t need to be:
             /// </summary>
             static void Main(string[] args)
             {
-                var parameters = new Dictionary<string, string>();
-                string secondaryConfigFile = string.Empty;
-                char[] delimiters = {'='};
-                foreach (string arg in args)
-                {
-                    string[] pieces = arg.Split(delimiters);
-                    if (pieces.Length == 2)
-                    {
-                        string key = pieces[0].Trim();
-                        string value = pieces[1].Trim();
-                        if (!string.IsNullOrWhiteSpace(key) && !string.IsNullOrWhiteSpace(value))
-                        {
-                            // Special case for SecondaryConfigFile
-                            if (key.Equals("SecondaryConfigFile", StringComparison.OrdinalIgnoreCase))
-                            {
-                                secondaryConfigFile = value;
-                            }
-                            else
-                            {
-                                parameters[key] = value;
-                            }
-                            continue;
-                        }
-                    }
-                    Console.WriteLine("Ignoring malformed input: {0}", arg);
-                };
-                Console.WriteLine(StartCommand.Execute(parameters secondaryConfigFile).ToString());
-                int autoFileId = 0;
-                while (true)
-                {
-                    Console.Write("Enter process ID to capture (blank to exit): ");
-                    string input = Console.ReadLine();
-                    if (input == string.Empty)
-                        break;
-                    if (!int.TryParse(input, out int processId))
-                    {
-                        Console.WriteLine("Not a valid int: " + input);
-                        continue;
-                    }
-                    Dictionary<string, string> snapshotParameters = new Dictionary<string, string>
-                    {
-                        { CommandConstStrings.TargetProcessId, input },
-                        { CommandConstStrings.OutputFile, autoFileId++.ToString() },
-                    };
-                    Console.WriteLine(SnapshotCommand.Execute(snapshotParameters).ToString());
+                string testAppPath = Path.GetFullPath("..\myApplication.exe");
+                string outputDir = Path.GetFullPath(".\TestOutput");
+                Process testProcess = Process.Start(testAppPath);
+                var config = Config.Builder.ForProcessId(testProcess.Id)
+                    .WithOutputFileFormat(OutputFileFormat.A11yTest)
+                    .WithOutputDirectory(outputDir)
+                    .Build();
+
+                var scanner = ScannerFactory.CreateScanner(config);
+
+                try {
```

Comment:
We should put opening brackets on new lines to be consistent with the rest of the code sample

Reply:
I should have commented in both places, this is also at the top in step 3 of 'how to run...'